src/openforms/js/components/admin/form_design/form-creation-form.js

```diff
@@ -845,7 +778,13 @@ function reducer(draft, action) {
           let {context: fieldPrefix, errors} = validationError;
           const _prefixedErrors = errors.map(err => {
             const fieldName = err.name.split('.')[0];
-            if (!tabsWithErrors.includes(fieldName) && FORM_FIELDS_TO_TAB_NAMES[fieldName]) {
+
+            if (fieldName === 'translations') {
```

Comment:
There are still things not going super smoothly with the confirmation tab. For example:
- Add a subject for the confirmation email template
- Add an invalid content
- Click on "Save form" twice in a row, without fixing the errors
=> The errors become duplicated:
![Screenshot from 2022-12-05 14-14-36](https://user-images.githubusercontent.com/19154114/205646425-121ac3cc-7d5a-437d-9ebc-93171ed7d622.png)